assignments/week_3/exercise_3_2/exercise_3_2.hpp

```diff
@@ -0,0 +1,31 @@
+#include <string>
```

Comment:
Did we cover hpp? also I don't see any implementation in here so why not just .h?

Reply:
In Codecademy they only ever use hpp